### ISSUE TITLE
Bug in `fixes`

### ISSUE THREAD

**Issue by janpaul123** [codecov/support#394]
Hi! I'm trying to use [fixes](https://docs.codecov.io/docs/fixing-paths) in an attempt to get around https://github.com/codecov/support/issues/377, but now I'm running into another issue. This is what the Codecov files look like in master now:

<img width="602" alt="screen shot 2017-04-17 at 15 14 31" src="https://cloud.githubusercontent.com/assets/177461/25106972/76050fe2-2381-11e7-8565-0feaf20667cd.png">

In reality, we have "app", "client", and "lib" in the "core" subdirectory, and "keystone" should be nested again in a "keystone" directory (e.g. "/keystone/keystone/file.py"). So I set up the following fixes in codecov.yml:

```
fixes:
  - "/app/::/core/app/"
  - "/client/::/core/client/"
  - "/lib/::/core/lib/"
  - "/keystone/::/keystone/keystone/"
```

However, now I get the following in Codecov (in list view):
<img width="849" alt="screen shot 2017-04-17 at 15 17 33" src="https://cloud.githubusercontent.com/assets/177461/25107006/a65662ae-2381-11e7-896c-5696cd7284f4.png">

Weirdly enough some files work, but some don't. Here are some that are correct:
<img width="614" alt="screen shot 2017-04-17 at 15 23 15" src="https://cloud.githubusercontent.com/assets/177461/25107032/ce57cfa4-2381-11e7-8301-eeb5bafaf7ed.png">

I wonder if maybe the fixes don't get applied correctly when combining. multiple reports from different CI instances (we run our tests in parallel). For your reference, you can look at https://codecov.io/gh/remix/remix/list/fdd0fc39ece0629b6f9e3660a200bf990ccb1495/

Thanks!

**Comment by janpaul123** [comment]
Ping to @stevepeak—any idea what might be causing this?

**Comment by stevepeak** [comment]
Hey, sorry of the delay.

The fixes added a double `//` which caused the issue. We resolved this recently and I believe your project is back on track.

Can you verify? 
Thank you!

**Comment by janpaul123** [comment]
It seems to still be happening. I made sure to create a new commit so Codecov ran fresh, and I still see the problem happening here: https://codecov.io/gh/remix/remix/list/7c28ac434bc249eee7b60f6d180fe8aedf831ad9/ (which is a commit that has the `fixes`)
Thanks for looking into it!

**Comment by janpaul123** [comment]
Cool, seems to work now, thanks! 👍
